timesketch/models/user.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This module implements the user model."""
+
+from flask_bcrypt import generate_password_hash
+from flask_bcrypt import check_password_hash
+from flask_login import UserMixin
+from sqlalchemy import Column
+from sqlalchemy import String
+from sqlalchemy.orm import relationship
+
+from timesketch.models import BaseModel
+
+
+class User(UserMixin, BaseModel):
+    """Implements the User model."""
+
+    username = Column(String(255), unique=True)
```

Comment:
It doesn't in this case. Here it is representing a username as a string in the database schema.